### ISSUE TITLE
цены для всех тикеров

### ISSUE THREAD

**Issue by jonny1960** [TinkoffCreditSystems/invest-openapi#521]
Можно сделать массив цен для всех тикеров в реальном времени, как это нормально сделано на бинансе "v3/ticker/price".

**Comment by AlexanderVolkovTCS** [comment]
Да, подобный метод, возвращающий last price по группе инструментов, уже запланирован

**Comment by jonny1960** [comment]
А когда можно его ожидать? Стопы через API уже пару лет ждем.

**Comment by Dimantchick** [comment]
Реализация стопов понятна. Собственно такая и ожидается. Просто нужны они, чтобы обеспечить большую скорость реакции и понизить вероятность пропуска, т.к. меньше вероятность ошибок из-за связи и подобного.

**Comment by Real3L0** [comment]
Я правильно понял, что это вопрос про подписку на несколько стаканов одновременно?
